src/lib/annotations/doc/README.md

```diff
@@ -0,0 +1,104 @@
+# Document and Presentation Annotations
+
+The document and presentation viewers supports highlight comment, highlight only, draw and point annotations.
+
+## Screenshot
+
+![Screenshot of document point annotations](../../../../images/doc_point.png)
+
+![Screenshot of document highlight annotations](../../../../images/doc_highlight.png)
+
+![Screenshot of document draw  annotations](../../../../images/doc_draw.png)
+
+## Supported File Extensions
+
+`doc, docx, odp, ods, odt, pdf, ppt, pptx`
+
+## Events
+See BoxAnnotations events.
+
+## Methods
+
+The following methods are available for the document viewer.
+
+| Method Name | Explanation | Method Parameters |
+| --- | --- | --- |
+| getAnnotatedEl | Determines the annotated element in the viewer | {HTMLElement} containerEl ||
+| getLocationFromEvent | Returns an annotation location on a document from the DOM event or null if no correct annotation location can be inferred from the event. For point annotations, we return the (x, y) coordinates and page the point is on in PDF units with the lower left corner of the document as the origin. For highlight annotations, we return the PDF quad points as defined by the PDF spec and page the highlight is on | {Event} DOM event, {String} annotation type ||
```

Comment:
Is this a method we expect people to call? Wouldn't they get the location from a create/delete event by default?